### ISSUE TITLE
Cannot collapse or expand feed categories with VoiceOver

### ISSUE THREAD

**Issue by l3land** [brentsimmons/NetNewsWire#5323]
Using NNW on an iPad or iPhone, it is impossible to expand or collapse feed categories in order to browse individual feeds within a folder. When voiceover is turned off, one can tap the right-facing arrow (“>”) to expand the feed list and browse each individually. When VoiceOver is enabled, however, one can merely browse the list of folders or enter the folder to see all the feeds collected together. What’s missing is an option to expand or collapse the folder. This would likely be done either through an action menu, which currently exists but only contains options for: delete, rename, drag item, or activate. This issue would be solved if “expand” and “collapse” were added to this action item list. Another slightly less optimal (because it would make browsing folder list less efficient) would be to make the “>” a clickable item that expands or collapses the folder feed list. 

I have never submitted an issue on GH before so I’m sorry if I’ve done this incorrectly! I love NNW and it’s tremendously accessible and you guys are awesome and long live FOSS!!!

**Comment by brentsimmons** [comment]
Fix coming in 7.1 👍

**Comment by l3land** [comment]
Wow awesome! I think this is the first time a human has responded to an accessibility request I’ve made and… it made my day!! Thank youuuu

**Comment by brentsimmons** [comment]
I’ll be counting on you to test it out and make sure it works as it should — and open another bug ticket here if it doesn’t! 😀

Also — please keep reporting accessibility issues when you find them. It’s a priority for us to make sure the app is completely accessible. It matters.
